src/server/pps/cmds/cmds.go

```diff
@@ -44,6 +44,32 @@ import (
 	"golang.org/x/net/context"
 )
 
+const (
+	// Singular variables are used for testing.
+	datum    = "datum"
+	job      = "job"
+	pipeline = "pipeline"
+	secret   = "secret"
+
+	// Plural variables are used below for user convenience.
+	datums    = "datums"
+	jobs      = "jobs"
+	pipelines = "pipelines"
+	secrets   = "secrets"
+
+	// Command 'verbs' are also used in testing.
+	create  = "create"
```

Comment:
if we aren't using these variables here I think it would make more sense to define them with the tests

Reply:
Sure, can do